tests/behat/features/fields.feature

```diff
@@ -183,6 +184,7 @@ Feature: Fields for Landing Page content type
     And I should see "Timelines" in the "select[name='field_landing_page_component[add_more][add_more_select]']" element
     And I should see "Complex Image" in the "select[name='field_landing_page_component[add_more][add_more_select]']" element
     And I should see "Promotion card" in the "select[name='field_landing_page_component[add_more][add_more_select]']" element
+    And I should see "Navigation card" in the "select[name='field_landing_page_component[add_more][add_more_select]']" element
```

Comment:
I think these tests should be extended to check the visibility of the conditional fields based on the user input.

These tests aren't using js so that wouldn't be possible in this scenario but one should be added that makes sure an editor is going to see the correct fields.

Reply:
@GROwen I have updated the PR with a more detailed check for these two paragraph fields.